client_log.go

```diff
@@ -27,16 +26,10 @@ type LogUploader interface {
 	Upload(api ApiRequester, server string, logs LogData) error
 }
 
-type LogEntry struct {
-	Timestamp string `json:"timestamp"`
```

Comment:
FYI, the API will try to parse Timestamp with default format for timestamps in JSON, that happens to be RFC3339.